src/frontend/js/copy-clipboard.ts

```diff
@@ -4,15 +4,7 @@ window.addEventListener("load", () => {
       const value = (currentTarget as HTMLElement).getAttribute(
         "data-copy-to-clipboard"
       );
-      const tmpInput = document.createElement("input");
-      tmpInput.type = "text";
-      document.body.appendChild(tmpInput);
-      tmpInput.value = value;
-      tmpInput.select();
-      tmpInput.setSelectionRange(0, 99999);
-
-      document.execCommand("copy");
-      document.body.removeChild(tmpInput);
+      navigator.clipboard.writeText(value);
```

Comment:
`navigator.clipboard` is [not supported on Internet Explorer](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard). @jnugh do you know if there is a way to tell Webpack to convert it in some way (if that's even possible)?

But today I learned that `document.execCommand("copy")` [is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) and shouldn't be used either (even though it continues to work on all modern browsers at the moment - but this might change in the future)... :unamused:

Reply:
So just for reference: For the moment, we decided to keep the deprecated `document.execCommand("copy")` solution and hope that the Browser support will continue for a while (ideally until we don't need to support IE anymore)